.github/workflows/greet-contributors.yml

```diff
@@ -1,23 +1,23 @@
-name: Greet Contributors
-
-on:
-  pull_request:
-    types:
-      - opened
-  issues:
-    types:
-      - opened
-
-jobs:
-  greet-contributor:
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2

```

Comment:
whats the need of deleting this and replacing it again with same script?

Reply:
@soumenkp2 I haven't removed and replaced it with the same script. I just renamed the `github` folder to `.github` (dot github). This change is made to designate the `.github` directory as the home for workflows, issue templates, pull request templates, and other project-specific files. These configurations should only take effect when the folder is named `.github`, and the permissions specified in the pull request are applied.